make.config.in

```diff
@@ -320,6 +321,7 @@ endif
 clean::
 	-@$(RM) -rf $(OBJ) $(DEPS) $(TARGET)
 	@for pp in $(DIRS); do echo "  " $$pp cleaned; $(MAKE) --no-print-directory -C $$pp clean; done
+	@rm -f $(SUB_LIBS)
```

Comment:
Looking a couple of lines earlier should this be `$(RM)` instead of straight `rm`?